docs/matchers/event.md

```diff
@@ -0,0 +1,19 @@
+# Event Matcher
+
+## Configuring opsdroid
+
+The `event` matcher matches all events of a certain type. It can, for example, be used to write a skill that responds to images or files.
+
+
+## Example
+
+```python
+from opsdroid.skill import Skill
+from opsdroid.matchers import match_event
+from opsdroid.events import Image, Message
+
+class ShoutyImagesSkill(Skill):
+    @match_event(Image)
+    async def loudimage(opsdroid, config, event):
```

Comment:
When using class based skills we do not take `opsdroid` and `config` in the methods. They are passed to the `__init__` of the `Skill` superclass and set as properties.